### PR TITLE
feat: increase command bar modal height

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Increases the `max-height` of the command bar dialog, so that more search results are visible, and command bar dialog contents have a little more breathing room.

## 📸 Design Screenshots

| Before | After |
| - | - |
| empty-before | empty-after |
| results-before | results-after |

## 🧪 Testing

- [ ] Visit any page with search enabled, such as [the home page][preview]
    - The search experience should continue to function as expected
    - The command bar dialog should be twice the max-height on the tallest viewports
    - On shorter viewports, the command bar dialog should continue to not overflow the viewport

## 💭 Anything else?

Not at the moment.

[task]: https://app.asana.com/0/1204333060736722/1204604641329015/f
[preview]: https://dev-portal-git-zssearch-increase-modal-height-hashicorp.vercel.app/

### CODE REVIEW

**Review by emilypersson1 — APPROVED** [review]
looks good! totally minor thing I noticed is when I'm not using a monitor on just my regular computer screen size, when I first open search (screenshot of the homepage, but happens on any page), the modal almost appears to be larger than the page just because of the global alert and the cookies alert. once those are dismissed it looks great and also looks great with a larger window with those alerts present. super minor edge case and not blocking, but just wanted to call it out!

Screenshot 2023-05-16 at 1 26 42 PM

File: src/components/dialog/dialog.module.css [hunk 1]
```diff
@@ -15,20 +15,17 @@
 }
 
 .contentWrapper {
-	align-items: flex-start;
+	align-items: center;
 	display: flex;
+	flex-direction: column;
```
[the remark this answers is not in the document]
Comment (kendallstrautman) [reply src/components/dialog/dialog.module.css:20]: Tested this on the bookmark dialog and it seems unaffected!

**Review by kendallstrautman — APPROVED** [review]
Tested on a few dialogs and at various viewport sizes 🚀  LGTM!

**Comment by heitzke** [comment]
Lookin good! It feels a touch on the tall side - can we shorten it a bit to be ~660 for the max-height? I think somewhere in there feels pretty good

**Comment by zchsh** [comment]
@heitzke I've updated to set the max-height to `660px` - let me know if that looks good to ship!

@emilypersson1 ah yeah great call around the global alert and consent manager making things feel a lil more cramped 🤔 I think Mike's tweak down to `660px` max-height should alleviate this on most viewports. On really short and wide viewports I think we'll still see this... We could in theory do more to try to account for those elements, and adjust the modal height accordingly, but that would be relatively high effort and intertwine some layout concerns... one note is it will never overflow the browser viewport, so that's something at least. Thanks for flagging, hopefully the recent height adjustment makes it feel acceptable, and if not, happy to dig deeper if y'all feel it would be worth it 🙇‍♂️

**Review by heitzke — APPROVED** [review]
Lookin' good to me!

**Pull request merged**